docs/pages/release_notes.md

```diff
@@ -1,6 +1,113 @@
+  * Supports both L1 and L2 Norms to calculate gradients
+* New tuned OpenCL BLAS backend,
+  [CLBlast](https://github.com/arrayfire/arrayfire/pull/1727).
+
+Improvements
+------------
+
+* Performance improvements when using \ref af::reorder()
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1766)</sup>
+* Improved unified backend performance
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1770)</sup>
+* ArrayFire now depends on Forge
+  v1.0. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1800)</sup>
+* Can now specify the FFT plan cache size using the
+  \ref af::setFFTPlanCacheSize() function.
+* Get the number of physical bytes allocated by the memory manager
+  \ref `af_get_allocated_bytes()`
+* \ref af::dot() can now return a scalar value to the host.
+
+Bug Fixes
+---------
+
+* Fixed improper release of default Mersenne random
+  engine. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1716)</sup>
+* Fixed assignment bug in CPU
+  backend. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1765)</sup>
+* Fixed memory leak in
+  \ref af_fir. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1765)</sup>
+* Fixing correctness of \ref af_pow for complex numbers by using Cartesian
+  form. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1765)</sup>
+* Corrected \ref af::select() with indexing in CUDA and OpenCL
+  backends. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1731)</sup>
+* Workaround for VS2015 compiler ternary
+  bug. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1771)</sup>
+* Fixed memory corruption in
+  `cuda::findPlan()`. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1793)</sup>
+* Argument checks in \ref af_create_sparse_array avoids inputs of type
+  int64. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1747)</sup>
+
+Build fixes
+-----------
+
+* On OSX, utilize new GLFW package from the brew package
+  manager. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1720)</sup>
+  <sup>[2](https://github.com/arrayfire/arrayfire/pull/1775)</sup>
+* Fixed CUDA PTX names generated by CMake v3.7.
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1689)</sup>
+
+Examples
+--------
+
+* New genetic algorithm example.
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1695)</sup>
+
+Documentation
+-------------
+
+* Updated `README.md` to improve readability and
+  formatting. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1726)</sup>
+* Updated `README.md` to mention Julia and Nim
+  wrappers. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1714)</sup>
+* Improved installation instructions -
+  `docs/pages/install.md`. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1740)</sup>
+
+Miscellaneous
+-------------
+
+* A few improvements for ROCm
+  support. <sup>[1](https://github.com/arrayfire/arrayfire/pull/1710)</sup>
+* Removed CUDA 6.5 support.
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1687)</sup>
+
+Known issues
```

Comment:
We can also mention that `fftconvolve` OpenCL test fail on Windows due to a driver bug. From our testing, we found that the bug persists in all 37*.xx series drivers.